spec/mysql2/client_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-RSpec.describe Mysql2::Client do
+RSpec.describe Mysql2::Client do # rubocop:disable Metrics/BlockLength
```

Comment:
Is there any chance to disable/extend this rubocop rule in specs only? rspec is usually bunch of huge blocks by design.